Makefile

```diff
@@ -37,7 +37,8 @@ MRLN=merlin
 TEST=tests
 DOCS=docs
 WKFW=merlin/examples/workflows/
-MAX_COMPLEXITY?=5
+MAX_COMPLEXITY?=10
+MAX_LINE_LENGTH=127
```

Comment:
fyi we've got another max line length in `setup.cfg`, so we should probably reconcile those.
https://github.com/LLNL/merlin/blob/9ea130dd83f67414a4759d331b944e5de3dc5ff7/setup.cfg#L14

Reply:
Have you thought of a way to do this?

Reply:
Yeah, I split the variables assigned at the top of the Makefile into a config.mk, and have that file and the CI draw from setup.cfg as the source of all configurable settings.